docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,171 @@
+- [Breaking changes, version 2.2 to 3.1 - .NET Core](../docs/core/compatibility/2.2-3.1.md)
+- [dotnet-trace tool - .NET Core](../docs/core/diagnostics/dotnet-trace.md)
+- [.NET Core SDK and runtime dependencies - .NET Core](../docs/core/install/dependencies.md)
+- [Check installed .NET Core versions on Windows, Linux, and macOS - .NET Core](../docs/core/install/how-to-detect-installed-versions.md)
+- [https://raw.githubusercontent.com/dotnet/docs/master/docs/core/install/includes/package-manager-heading-hack-pkgname.md](../docs/core/install/includes/package-manager-heading-hack-pkgname.md)
+- [Install .NET Core on CentOS 7 - package manager - .NET Core](../docs/core/install/linux-package-manager-centos7.md)
+- [Install .NET Core on Debian 10 - package manager - .NET Core](../docs/core/install/linux-package-manager-debian10.md)
+- [Install .NET Core on Debian 9 - package manager - .NET Core](../docs/core/install/linux-package-manager-debian9.md)
+- [Install .NET Core on Fedora 29 - package manager - .NET Core](../docs/core/install/linux-package-manager-fedora29.md)
+- [Install .NET Core on Fedora 30 - package manager - .NET Core](../docs/core/install/linux-package-manager-fedora30.md)
+- [Install .NET Core on openSUSE 15 - package manager - .NET Core](../docs/core/install/linux-package-manager-opensuse15.md)
+- [Install .NET Core on Linux RHEL 7 package manager - .NET Core](../docs/core/install/linux-package-manager-rhel7.md)
+- [Install .NET Core on Linux RHEL 8.1 package manager - .NET Core](../docs/core/install/linux-package-manager-rhel81.md)
+- [Install .NET Core on SLES 12 - package manager - .NET Core](../docs/core/install/linux-package-manager-sles12.md)
+- [Install .NET Core on SLES 15 - package manager - .NET Core](../docs/core/install/linux-package-manager-sles15.md)
+- [Install .NET Core on Ubuntu 16.04 package manager - .NET Core](../docs/core/install/linux-package-manager-ubuntu-1604.md)
+- [Install .NET Core on Ubuntu 18.04 package manager - .NET Core](../docs/core/install/linux-package-manager-ubuntu-1804.md)
+- [Install .NET Core on Ubuntu 19.04 package manager - .NET Core](../docs/core/install/linux-package-manager-ubuntu-1904.md)
+- [Install .NET Core runtime on Windows, Linux, and macOS - .NET Core](../docs/core/install/runtime.md)
+- [Install .NET Core SDK on Windows, Linux, and macOS - .NET Core](../docs/core/install/sdk.md)
+- [Breaking changes for WinForms - .NET Framework to .NET Core](../docs/core/porting/winforms-breaking-changes.md)
+- [Garbage collector config settings](../docs/core/run-time-config/garbage-collector.md)
+- [Run-time config](../docs/core/run-time-config/index.md)
+- [dotnet run command](../docs/core/tools/dotnet-run.md)
+- [dotnet sln command](../docs/core/tools/dotnet-sln.md)
+- [What's new in .NET Core 3.0](../docs/core/whats-new/dotnet-core-3-0.md)
+
+## C# language
+
+- [https://raw.githubusercontent.com/dotnet/docs/master/docs/csharp/index.yml](../docs/csharp/index.yml)
```

Comment:
This doesn't seem like it belongs in this list.

Reply:
same bug.